[2025-04-04T13:12:56.808+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-04T13:12:19.705762+00:00 [queued]>
[2025-04-04T13:12:56.815+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-04T13:12:19.705762+00:00 [queued]>
[2025-04-04T13:12:56.816+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-04T13:12:56.828+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Write_data_in_kafka> on 2025-04-04 13:12:19.705762+00:00
[2025-04-04T13:12:56.832+0000] {standard_task_runner.py:60} INFO - Started process 5742 to run task
[2025-04-04T13:12:56.837+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_Proj', 'Write_data_in_kafka', 'manual__2025-04-04T13:12:19.705762+00:00', '--job-id', '201', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks.py', '--cfg-path', '/tmp/tmpo99bzd7j']
[2025-04-04T13:12:56.838+0000] {standard_task_runner.py:88} INFO - Job 201: Subtask Write_data_in_kafka
[2025-04-04T13:12:56.877+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_Proj.Write_data_in_kafka manual__2025-04-04T13:12:19.705762+00:00 [running]> on host 7fc40170f80d
[2025-04-04T13:12:56.947+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Proj' AIRFLOW_CTX_TASK_ID='Write_data_in_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T13:12:19.705762+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-04T13:12:19.705762+00:00'
[2025-04-04T13:12:57.954+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-04T13:12:57.969+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Weather_Proj, task_id=Write_data_in_kafka, execution_date=20250404T131219, start_date=20250404T131256, end_date=20250404T131257
[2025-04-04T13:12:58.013+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-04T13:12:58.044+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
